packages/atomic/dev/examples/genqa.html

```diff
@@ -27,6 +29,17 @@
 
       // Trigger a first search
       searchInterface.executeFirstSearch();
+
+      const advQueriesActions = loadAdvancedSearchQueryActions(searchInterface.engine);
+
+      searchInterface.engine?.dispatch(
+        advQueriesActions.updateAdvancedSearchQueries({
+          aq: '@fileType=html',
+          cq: '@fileType=html',
+          dq: '',
+          lq: '',
+        })
+      );
```

Comment:
Considering this change is very specific to the issue you fixed, I would not commit it. You could instead document how to load and dispatch an Headless action in the test page.